### PR TITLE
change to standard-airbnb

### PR DESCRIPTION
**Pull request by dakebl**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -17,29 +14,23 @@ module.exports = {
...
+    'sort-destructure-keys',
+    'node'
   ],
   rules: {
     'brace-style': ['error', 'stroustrup'],
     'comma-dangle' : [2, 'always-multiline'],
     'curly': [2, 'multi-line', 'consistent'],
     'no-unused-vars': [2, { 'ignoreRestSiblings': false }],
     'node/no-deprecated-api': 1,
-    'react-hooks/rules-of-hooks': 'error',
```
Comment (wezlar) [inline .eslintrc.js:42]: Why was this taken out? Is it part of Airbnb already? Or was there a different reason?

Comment (dakebl) [reply .eslintrc.js:42]: Yeah, it's already there: https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/rules/react-hooks.js#L15

**Comment by JayboBaker** [comment]
@dakebl are you game to merge this as we're fixing some other lint bits in FE at the moment...

**Comment by inbn** [comment]
It looks like [eslint-config-airbnb-standard](https://www.npmjs.com/package/eslint-config-airbnb-standard) wants semicolons:

> It uses original Airbnb Style config extended with JavaScript Standard Style config (with semicolons).

I'm going to add this and see what happens

```js
"rules": {
  "semi": ["error", "never"]
}
```

**Pull request merged**